idr0007-srikumar-sumo/idr0007-study.txt

```diff
@@ -43,7 +44,7 @@ Screen Number	1
 Comment[IDR Screen Name]	idr0007-srikumar-sumo/screenA																																
 Screen Description	"In this screen two mutant yeast strains unable to synthesize SUMO chains (pro-smt3allR and smt3allR), were crossed with 384 yeast strains expressing GFP-tagged proteins (Huh et al., 2003) previously linked to the DNA damage response (Tkach et al., 2012).  144 GFP-tagged proteins displayed a change in localization and/or intensity when expressed in the smt3allR mutant grown in rich medium."																																
 Screen Size	Plates: 12	5D Images: 3456	Planes: 6912	Average Image Dimension (XYZCT): 1346 x 1003 x 1 x 2 x 1	Total Tb: 0.2																												
-Screen Example Images	http://idr-demo.openmicroscopy.org/webclient/?show=well-37013	http://idr-demo.openmicroscopy.org/webclient/img_detail/95543/	"WT plate1;G13"																														
+Screen Example Images	http://idr-demo.openmicroscopy.org/webclient/?show=well-37013	http://idr-demo.openmicroscopy.org/webclient/img_detail/95543/	WT plate1;G13																														
```

Comment:
https

Reply:
There will be lots of cases where we need to change http://idr-demo.openmicroscopy.org to https://idr.openmicroscopy.org.  I could go through them all now, or do as a separate PR.

Reply:
```
$ git grep idr-demo | wc
      25    1718   17936
```

I would say separate PR please else this is becoming a moving target.